### PR TITLE
chore: fix missing imports

### LINKED ISSUE #103: Non existing imported class
As been discussed in PR #101 

Some of the classes imports class ( `Illuminate\Foundation\Application`) from non existing package in `composer.json`, we should require `illuminate\contracts` and resolve the class using DI.

### PR DESCRIPTION
**Pull request by veelasky**
Fix missing import from package as been discussed in #101 and #103

will resolve #103

### CODE REVIEW

**Comment by veelasky** [comment]
I'll mark this as draft, and make changes in accordance with #101

**Comment by arkaitzgarro** [comment]
@veelasky Composer file has been updated in master, you can now update your branch accordingly.

File: composer.json [hunk 1]
```diff
@@ -28,7 +29,7 @@
         "codeception/mockery-module": "^0.4.0",
         "dms/phpunit-arraysubset-asserts": "^0.1.0",
         "friendsofphp/php-cs-fixer": "^2.16",
-        "orchestra/testbench": "^4.0 || ^5.0 || ^6.0",
+        "orchestra/testbench": "^3.5",
```
Comment (arkaitzgarro) [inline composer.json]: It looks like this change should not be here. Also tests are not running, did you merge changes from master?

**Comment by arkaitzgarro** [comment]
@veelasky sorry, but now file changes are missing :)

**Comment by veelasky** [comment]
@arkaitzgarro just did what you said and made sure that I was merging from master.

**Comment by arkaitzgarro** [comment]
> @arkaitzgarro just did what you said and made sure that I was merging from master.

Looks like the merge didn't go really well on your side, the original changes to use Laravel contracts are not present anymore.

**Review by arkaitzgarro — APPROVED** [review]

**Comment by veelasky** [comment]
my badd....

**Pull request merged**